src/app/dropdown/dropdown-angular-close-item-false.demo.html

```diff
@@ -10,14 +10,14 @@
             <clr-icon shape="error" class="is-error" size="24"></clr-icon>
             <clr-icon shape="caret down"></clr-icon>
         </button>
-        <div class="dropdown-menu">
+        <clr-dropdown-menu>
```

Comment:
I know it may seem redundant but with the wizard I was adding the component tag as a classname (like `class="clr-dropdown-menu"`) so we (and end users) could have a hook to styling that was not reliant on the component's tagname.